specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.2/examples/AnalyzeBatch.json

```diff
@@ -0,0 +1,16 @@
+{
+  "parameters": {
+    "endpoint": "{endpoint}",
+    "Content-Type": "application/json",
+    "Ocp-Apim-Subscription-Key": "{API key}",
+    "modelId": "{modelId}",
+    "body": {}
+  },
+  "responses": {
+    "202": {
+      "headers": {
+        "Operation-Location": "{endpoint}/formrecognizer/v2.1-preview.1/custom/models/{modelId}/analyzeResults/{resultId}"
```

Comment:
Please update version across all examples.

Reply:
The version is already updated. The PR contains multiple commits, some of them are auto generated. The initial commit (created by a tool) simply copy everything from preview.1, the second commit (also created by the tool) updated all version numbers to preview.2